packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-federated-sign-in-button/amplify-federated-sign-in-button.component.html

```diff
@@ -8,7 +8,7 @@
 >
   <div
     class="amplify-flex federated-sign-in-button-row"
-    style="flex-direction: row; justify-content: center"
+    style="flex-direction: row; justify-content: center; align-items: center"
```

Comment:
This fixed an issue where the federated icons were slightly above the text on the line